normal/gmail.profile

```diff
@@ -0,0 +1,83 @@
+# Make requests look like GMail web requests
+#
+# Author: @ChrisTruncer
+
+https-certificate {
+	set CN       "gmail.com";
+	set O        "Google's GMail";
+	set C        "US";
+	set L        "Mountain View";
+	set OU       "Google Mail";
+	set ST       "CA";
+	set validity "365";
+}
+
+http-get {
+	set uri "/mail/u/1/";
+	client {
+		header "Content-Type" "application/octet-stream";
+
+		metadata {
+			base64;
+			prepend "OSID=";
+			header "Cookie";
+		}
+
+		parameter "utmcn" "1";
+		parameter "utmcs" "ISO-8859-1";
+		parameter "utmsr" "1280x1024";
+		parameter "utmsc" "32-bit";
+		parameter "utmul" "en-US";
+
+	}
+
+	server {
+		header "X-Content-Type-Options" "nosniff";
+		header "X-Frame-Options" "SAMEORIGIN";
+		header "Cache-Control" "no-cache, no-store, max-age=0, must-revalidate";
+		header "X-XSS-Protection" "1; mode=block";
+		header "Server" "GSE";
+		output{
+			print;
+		}
+	}
+}
+
+http-post {
+	set uri "/mail/u/0/";
+	client {
```

Comment:
A Content-Type header from the client would be good as you're POSTing data back to the server. I recommend: application/x-www-form-urlencoded